projects/plugins/beta/src/class-plugin.php

```diff
@@ -575,8 +578,13 @@ public function dev_upgrader_response() {
 		} elseif ( 'master' === $dev_info->source && isset( $manifest->master ) &&
 			Semver::greaterThan( $manifest->master->version, $dev_info->version )
 		) {
-			// Master has been updated.
+			// Master has been updated, if master is the main branch.
 			list( , $info ) = $this->get_which_and_info( 'master', '' );
+		} elseif ( 'trunk' === $dev_info->source && isset( $manifest->trunk ) &&
+		Semver::greaterThan( $manifest->trunk->version, $dev_info->version )
```

Comment:
Nitpick, bad indentation
```suggestion
			Semver::greaterThan( $manifest->trunk->version, $dev_info->version )
```